spec/models/web_search_spec.rb

```diff
@@ -334,44 +338,77 @@
 
       context 'when we want X Bing/Google results from page Y and there are 0 <= n < X of them' do
         let(:search) do
-          described_class.new(query: 'odie backfill page 2', affiliate: affiliate, page: 2)
+          described_class.new(query: 'odie backfill', affiliate: affiliate, page: page)
         end
+        let(:page) { 1 }
 
         before do
           ElasticIndexedDocument.recreate_index
 
           bing_api_url = "#{BingV6WebSearch::API_HOST}#{BingV6WebSearch::API_ENDPOINT}"
-          page2_6results = Rails.root.join('spec/fixtures/json/bing_v6/web_search/page2_6results.json').read
-          stub_request(:get, /#{bing_api_url}.*odie backfill page 2/).
-            to_return(status: 200, body: page2_6results)
+          page1_6results = Rails.root.join('spec/fixtures/json/bing_v6/web_search/page1_6results.json').read
+          stub_request(:get, /#{bing_api_url}.*odie backfill/).
+            to_return(status: 200, body: page1_6results)
         end
 
         context 'when the affiliate has social image feeds and there are Odie results' do
           before do
-            affiliate.indexed_documents.create!(
-              title: 'odie backfill page 2', description: 'odie backfill page 2',
-              url: 'http://nonsense.gov', last_crawl_status: IndexedDocument::OK_STATUS
-            )
+            21.times do |index|
+              affiliate.indexed_documents.create!(
+                title: "odie backfill #{index}",
+                description: "odie backfill #{index}",
+                url: "http://nonsense.gov/#{index}",
+                last_crawl_status: IndexedDocument::OK_STATUS
+              )
+            end
             ElasticIndexedDocument.commit
             allow(affiliate).to receive(:has_social_image_feeds?).and_return true
           end
 
-          it 'indicates that there is another page of results' do
-            search.run
-            expect(search.total).to be >= 20
-            expect(search.results.size).to be == 6
-            expect(search.startrecord).to be == 11
-            expect(search.endrecord).to be == 16
+          context 'when returning the first page with commercial results' do
+            it 'indicates via the search.total that there is another page of results' do
+              search.run
```

Comment:
I think `search.run` could be moved to a `before` block in this `context`.

Reply:
Done.